src/reporters/utils/helpers.js

```diff
@@ -42,7 +43,11 @@ exports.getTestInfo = test => {
     };
 
     if (test.err) {
-        testInfo.error = getTestError(test);
+        testInfo.error = chalk.supportsColor ? getTestError(test) : stripAnsi(getTestError(test));
+    }
+
+    if (test.err && test.err.snippet) {
+        testInfo.errorSnippet = chalk.supportsColor ? test.err.snippet : stripAnsi(test.err.snippet);
```

Comment:
This way we can use colorfull version for console, and remove control characters if user pipes logs into file